.travis.yml

```diff
@@ -16,8 +16,9 @@ jobs:
       install:
       script: |
         clang-format-7 --version
-        git-clang-format-7 --binary clang-format-7 "${TRAVIS_BRANCH}"
-        git diff > formatted.diff
+        cat .clang-format-ignore | while read file ; do EXCLUDES+="\':\(top,exclude\)$file\' " ; done
+        git-clang-format-7 --binary clang-format-7 "${TRAVIS_BRANCH}" -- $EXCLUDES
```

Comment:
My `git-clang-format-6.0 --help` output suggests that listing file names after `--` would mean that _only_ those files are checked. Has this changed in `git-clang-format-7`? But then there's the `(top,exclude)` magic being added, where is that documented?

Reply:
Thanks!